std/parallelism.d

```diff
@@ -2864,13 +2864,13 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref T opIndex(size_t index) const
```

Comment:
That would be the case in C++, not in D. In D if A contains B then const A contains const B.